hw/ip/rom_ctrl/data/rom_ctrl.hjson

```diff
@@ -161,6 +161,7 @@
           name: "ROM"
           items: "8192" // 32 KiB
           swaccess: "ro",
+          data-intg-passthru: "true",
```

Comment:
I would probably prefer to merge https://github.com/lowRISC/opentitan/pull/9539 before making this change. That PR has been waiting for review for a week and fixes the RTL bug properly.

Reply:
Thanks!